tools/xcconfigs/LLVM.xcconfig

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Damn this came back. I was trying to make it a blank file that git would keep in the repo but ignore changes. The Xcode build system fails if the file isn't there because other xcconfig files `#include` it and it doesn't run the script to generate this xcconfig file if any of the xcconfig files have any errors. I made my git ignore changes to it but that only works locally for myself. Not sure what would work for any contributors. Any suggestions?

Reply:
Also I am running 6.0.1 and planning on trying out 7.0.0rc2 soon so it will change lots if we can't just ignore it.

Reply:
>.>